src/core/thread/mle_router.hpp

```diff
@@ -325,6 +325,22 @@ class MleRouter : public Mle
      */
     void SetRouterDowngradeThreshold(uint8_t aThreshold) { mRouterDowngradeThreshold = aThreshold; }
 
+    /**
+     * This method returns the MLE_CHILD_ROUTER_LINKS value.
+     *
+     * @returns The MLE_CHILD_ROUTER_LINKS value.
+     *
+     */
+    uint8_t GetChildRouterLinks(void) const { return mChildRouterLinks; }
+
+    /**
+     * This method sets the MLE_CHILD_ROUTER_LINKS value.
+     *
+     * @param[in]  aChildRouterLinks  The MLE_CHILD_ROUTER_LINKS value.
+     *
+     */
+    void SetChildRouterLinks(uint8_t aChildRouterLinks) { mChildRouterLinks = aChildRouterLinks; }
```

Comment:
Do we need to check whether current active link count is less than `aChildRouterLinks` here ?

Reply:
Yes. Adding `    VerifyOrExit(instance.Get<Mle::MleRouter>().IsDisabled(), error = kErrorInvalidState);` to only allow param change when device is not running/attached.

Reply:
done.